### PR TITLE
Add test fixture

### PR DESCRIPTION
**Pull request by abbymapes**

### CODE REVIEW

File: walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java [hunk 1]
```diff
@@ -29,64 +29,85 @@
...
 
   Collection<TimeRange> possibleTimes = new ArrayList<>();
-  ArrayList<TimeRange> allConflicts = new ArrayList<>();
-  ArrayList<TimeRange> conflicts = new ArrayList<>();
+  //ArrayList<TimeRange> allConflicts = new ArrayList<>();
+  //ArrayList<TimeRange> conflicts = new ArrayList<>();
   long requestDuration;
-  Collection<String> attendees;
-  
+  //Collection<String> attendees;
```
Comment (dibenede) [inline walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:35]: delete commented out stuff

File: walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java [hunk 2]
```diff
@@ -29,64 +29,85 @@
...
+    allAttendees.addAll(mandatoryAttendees);
+
+    Collection<TimeRange> possibleTimes = new ArrayList<>();
+
     if (requestDuration > WHOLE_DAY.duration()){
       return possibleTimes;
     }
     
-    if (attendees.size() == 0){
+    if (attendees.size() == 0 && optionalAttendees.size() == 0){
```
Comment (dibenede) [inline walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:52]: prefer .empty() to size() == 0

Comment (dibenede) [reply walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:52]: below lines, as well

File: walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java (hunk 1 above)
Comment (abbymapes) [reply walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:35]: fixed in next push

File: walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java (hunk 2 above)
Comment (abbymapes) [reply walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:52]: fixed in next push

**Pull request merged**